internal/commands/push/command_test.go

```diff
@@ -581,12 +618,12 @@ Successfully pushed app up: eggcorn-abcde
 
 			assert.Nil(t, cmd.Handler(nil, ui, cli.Clients{Realm: realmClient}))
 			assert.Equal(t, `Determining changes
-Transpiled dependency sources
 Creating draft
 Pushing changes
 Deploying draft
 Deployment complete
-Uploaded dependencies archive
+Uploaded dependencies
```

Comment:
yeah, so to your question earlier I think we should definitely remove this to avoid the double-printing of "dependencies" messages